### ISSUE TITLE
Persistent and stable labelling and linking of URI's

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-taf/conjoin#26]
The assessment outputs from OHAT are presented via the OSPAR Assessment Portal (OAP), with an embedded link which allows users to directly interact with 
OHAT: https://oap.ospar.org/en/ospar-assessments/committee-assessments/hazardous-substances-and-eutrophication/mime/cemp-levels-and-trends-marine-contaminants/cemp-2021/ It is important that we maintain the “assessmentperiod=” portion of the URL, from the above example: 

https://dome.ices.dk/ohat/?assessmentperiod=2021 to enable persistent links to the appropriate assessment period data.

**Comment by HansMJ** [comment]
If we want persistent links I suggest creating DOI's per assessment instance

**Comment by neil-ices-dk** [comment]
> If we want persistent links I suggest creating DOI's per assessment instance

yes, i think that would be straightforward (the final assessment instance).

**Comment by neil-ices-dk** [comment]
also refer to acknowledgement i.e. readme or iso 19115 that describes value chain as needed
